apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx

```diff
@@ -44,9 +45,15 @@ interface ManagerDashboardProps {
   userQuery: FragmentType<typeof ManagerDashboardUser_Fragment>;
 }
 
+type AccordionItems = ("your_talent_searches" | "")[];
+
 const ManagerDashboard = ({ userQuery }: ManagerDashboardProps) => {
   const paths = useRoutes();
   const intl = useIntl();
+  const [accordionItems, setAccordionItems] = useLocalStorage(
+    "accordion_items",
+    ["your_talent_searches"],
```

Comment:
This doesn't seem to be stored on initial page load.  It only gets stored after you interact with it.  I guess it is smart enough to know that it doesn't need to store the default value because it works.  :shrug:

Reply:
When the accordionItems value changes it shows up in local storage, so before that event it doesn't show up. The initial values are added to the Accordion.Root so it should be steady.